app/models/bgs_dependents/veteran.rb

```diff
@@ -52,5 +52,19 @@ def veteran_response(participant, va_file_number, address, end_product)
         benefit_claim_type_end_product: end_product
       }
     end
+
+    private
+
+    def marital_status(dependents_application)
+      return 'Never Married' if dependents_application.dig('veteran_was_married_before') == false
+
+      if dependents_application.dig('does_live_with_spouse', 'spouse_does_live_with_veteran') == false
+        return 'Separated'
+      end
+
+      return 'Married' if dependents_application.dig('does_live_with_spouse', 'spouse_does_live_with_veteran') == true
+
+      nil
```

Comment:
```suggestion
      spouse_lives_with_vet = dependents_application.dig('does_live_with_spouse', 'spouse_does_live_with_veteran')
      return nil if spouse_lives_with_vet.nil?
      spouse_lives_with_vet ? 'Married' : 'Separated'
```

No need to call dig twice. I think the logic can be cleaned up a bit as well.